### DISCUSSION TITLE
ability to see traffic for each customer

### DISCUSSION THREAD

**Discussion by Julieng6v** [wg-easy/wg-easy#1104] [Ideas]
Hello

This would be great if the total client traffic was undisclosed and when we restart the server it remains and does not start from 0

![image](https://github.com/wg-easy/wg-easy/assets/73670804/f84f6c69-fadf-4240-bb15-fa05f5d6ea31)

**Comment by eandersons** [comment]
There are similar issues and discussions regarding this. For example: #1037, #1011, #732 (somewhat related).

In short - WireGuard does not persist connection information after restart, and if you want for connection details and traffic to accumulate, then you either have to use third party tools or you are free to implement this in WireGuard Easy and create a pull request.

**Reply by Julieng6v** [reply]
Okay thank you but do you know any extensions to store data?
